OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoLogic.cs

```diff
@@ -9,6 +9,7 @@
  */
 #endregion
 
+using System;
```

Comment:
Stupid question - is this import needed? I don't see anything referencing it directly in the changes here.